recipe/bld.bat

```diff
@@ -1,23 +1,23 @@
-mkdir build
-if errorlevel 1 exit 1
+mkdir build && cd build
 
-cd build
-if errorlevel 1 exit 1
+set CMAKE_CONFIG=Release
 
-cmake .. ^
-        -G %CMAKE_GENERATOR% ^
-        -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
-        -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
-        -DBUILD_STATIC_LIBS=1 ^
-        -DBUILD_SHARED_LIBS=1 ^
-        -DPYTHON_EXECUTABLE="%PYTHON%"
-if errorlevel 1 exit 1
+rem shared libs fail to build, built static instead
+rem https://github.com/open-source-parsers/jsoncpp/issues/631
 
-cmake --build . --config Release
+cmake ^
+  -G "NMake Makefiles" ^
+  -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+  -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+  -DCMAKE_BUILD_TYPE="%CMAKE_CONFIG%" ^
+  -DBUILD_STATIC_LIBS=OFF ^
+  -DBUILD_SHARED_LIBS=ON ^
```

Comment:
That's right. NVM.

Some people prefer to build with static libraries. Admittedly very infrequent with packages in conda-forge. Though there are a few cases (e.g. `python` uses the static library from `bzip2`). Mainly they are useful for people building single executables outside of conda-forge.